src/Bundle/Resources/config/services/state.xml

```diff
@@ -12,65 +12,29 @@
 -->
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <imports>
+        <import resource="state/processor.xml" />
+        <import resource="state/provider.xml" />
+    </imports>
```

Comment:
You can put the generic import in the `services.xml` file
```
<imports>
    <import resource="services/**/*.xml" />
</imports>
```

Reply:
you're right, done (with a few change on your proposal).